api/src/main/java/io/smallrye/stork/api/config/ServiceDiscoveryAttributes.java

```diff
@@ -0,0 +1,15 @@
+package io.smallrye.stork.api.config;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface ServiceDiscoveryAttributes {
```

Comment:
I don't understand very well the added value of this layer (see my comment in ServiceDiscoveryName annotation), why not having something like:
```
public @interface ServiceDiscoveryName { //to rename to ServiceDiscoveryProvider??
    String type();

    ServiceDiscoveryAttribute[] attributes() default {};
}
```

Reply:
this is just for users to be able to put multiple annotations on a class.
Without a class like this, only a single one can be added. I know, that's weird :)